docs/asset-store/assets/example-openapi-service.yaml

```diff
@@ -0,0 +1,124 @@
+openapi: 3.0.0
+info:
+  description: This example shows API requirements for a service that communicates with the Asset Store. This example contains `/convert`, `/validate`, and `/extract` endpoints.
+  title: Example OpenAPI service
+  license:
+    name: Apache 2.0
+    url: https://www.apache.org/licenses/LICENSE-2.0
+  version: 0.0.1
+paths:
+  /convert:
+    post:
+      description: Converts files sent in the `multipart/form-data` form from the old AsyncAPI version to 2.0.0.
+      tags:
+        - convert
+      summary: Converts the AsyncAPI specification to version 2.0.0.
+      operationId: convertReq
+      requestBody:
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              properties:
+                parameters:
+                  description: Additional parameters for the service
+                  type: string
+                content:
+                  description: File to convert
+                  type: string
+                  format: binary
+      responses:
+        "200":
+          $ref: "#/components/responses/successConversionResponse"
+        "304":
+          $ref: "#/components/responses/notModified"
+  /validate:
+    post:
+      description: Validates files sent in the `multipart/form-data` form against the AsyncAPI schema in version 2.0.0.
+      tags:
+        - validate
+      summary: Validates the AsyncAPI specification against the AsyncAPI schema in version 2.0.0.
+      operationId: validateReq
+      requestBody:
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              properties:
+                parameters:
+                  description: Additional parameters for the service
+                  type: string
+                content:
+                  description: File to validate
+                  type: string
+                  format: binary
+      responses:
+        "200":
+          $ref: "#/components/responses/successValidationResponse"
+        "422":
+          $ref: "#/components/responses/unprocessableEntity"
+  /extract:
+    post:
+      description: Extracts metadata from text files sent in the `multipart/form-data` form. As a field name, specify the relative or absolute path to the file.
+      tags:
+        - extract
+      summary: Extracts metadata from files.
+      operationId: extractReq
+      requestBody:
+        content:
+          multipart/form-data:
+            schema:
+              type: object
+              additionalProperties:
+                type: string
+                format: binary
+      responses:
+        "200":
+          $ref: "#/components/responses/successResponse"
+servers:
```

Comment:
why do we need it, why not just baseURL?

Reply:
We discuss with @kazydek to remove it from spec :)

Reply:
@derberg If I were to change `url` into `baseURL` what link would you provide here?

Reply:
sorry, I introduced some confusion here, sorry for that
baseUrl (actually host and basePath) is a thingy for swagger 2.0 
in 3.0 it is solved totally differently https://swagger.io/specification/#server-object-example
so what you have now is good, I would not change that, sorry again